src/ImpactEvaluator.sol

```diff
@@ -79,56 +80,29 @@ contract ImpactEvaluator is AccessControl, Nonces {
             addresses.length == scores.length,
             "Addresses and scores length mismatch"
         );
-        require(roundIndex < currentRoundIndex, "Round not finished");
-
-        (uint indexInOpenRounds, Round storage round) = getOpenRound(
-            roundIndex
+        require(roundIndex < currentRound.index, "Round not finished");
+        require(
+            roundIndex == previousRound.index && !previousRound.scored,
+            "Open round does not exist"
```

Comment:
this error message is a bit confusing. Can we be more clear with what the issue is here?

Reply:
Improved, wdyt?